etest/docker/container.py

```diff
@@ -23,8 +23,16 @@ def commit(container: Container, tag: str, repository: str, *args, **kwargs):
     return repo + ":" + tag
 
 
-def create(overlay: str, *args, **kwargs):
-    """Create a Docker container."""
+def create(overlay: str = None, *args, **kwargs):
```

Comment:
@alunduil let me know what you think of this approach.

Reply:
I think that's fine, but the type on overlay is incorrect. ;)

Reply:
you're right, it should be a Path

Reply:
I was more thinking that by having an assignment to None it has to be Optional[str], but Optional[Path] is even better.